Styling for the capture view

Lindayezou opened the pull request.

Commit:
Refactors code to accommodate dice png in the capture page

Changes pokemon model, data generator function and pokemon data to generate and hold dice png information
Renders appropriate scaled sized die in the capture view

 Changes to be committed:
	modified:   capturePokemon/capturePokemon.html
	modified:   capturePokemon/capturePokemon.js
	modified:   style.css
	modified:   ../server/data/createDataFunctions.js
	modified:   ../server/data/pokemonData.js
	modified:   ../server/pokemon/pokemonModel.js

Commit:
Resizes pokemon and dice

 Changes to be committed:
	modified:   capturePokemon/capturePokemon.html
	modified:   style.css

Commit:
Styles capture page

Implements dice instead of numbers
Pokemon image and data orgianized in box
Results message pops up below the box

 Changes to be committed:
	modified:   capturePokemon/capturePokemon.html
	modified:   style.css

Commit:
Decreases size of pokedex header

 Changes to be committed:
	modified:   capturePokemon/capturePokemon.html

hiteshlala commented:
completes #135

hiteshlala commented on public/capturePokemon/capturePokemon.js:
```diff
@@ -27,7 +27,9 @@ angular.module('pokemon.capture', [])
 
         gameFactory.getAvailablePokemon($scope.gameId, $scope.currentTurnPlayerId)
           .then(function(pokemon) {
+            console.log(pokemon);
```
delete console.log

hiteshlala commented on server/data/createDataFunctions.js:
```diff
@@ -124,5 +133,5 @@ var makeUnique = function() {
   // return result;
 };
-// console.log(makeUnique());
+console.log(makeUnique());
```
comment this line out so it does not run when file is run

Commit:
Deletes and comments out various console.logs

 Changes to be committed:
	modified:   capturePokemon/capturePokemon.js
	modified:   ../server/data/createDataFunctions.js
